### ISSUE TITLE
[BUG] Valve Index base station update

### ISSUE THREAD

**Issue by jug007** [ValveSoftware/SteamVR-for-Linux#294]
**Describe the bug**
I am not able to update the Valve Index base stations firmware. Bluetooth isn't implemented and USB doesn't make the lighthouse show up in SteamVR Update Devices. I am running SteamVR beta 1.10.6.
I am also not finding any clear instructions on the internet, on how to update the device.

**To Reproduce**
1. Connect Lighthouse to computer via usb
2. Connect Lighthouse power
3. open SteamVR
4. got to Devices -> Update Devices
5. the device will not show up

**Expected behavior**
There should be an option to update the Lighthouse, or information on the process of updating, in case this is the wrong procedure.

**System Information (please complete the following information):**
- Distribution: Linux Mint 19.3 (Ubuntu 18.04.3 based)
- SteamVR version: beta 1.10.6
- Steam client version: Jan 23 2020 23:59:24
- Opted into Steam client beta?: No
- Graphics driver version:: 440.48.02
- Gist for SteamVR System Information: https://gist.github.com/jug007/1ad7a0760b5f979c53c7a32ed656f77b

**Additional context**
The reason I want to update the lighthouse is an issue I have encoutered, which makes the lighthouse useless for tracking compared to the other one. It seems like it is not working at all and I am using just one lighthouse instead of both. The lighthouse makes the distinctive sound that they make and two strong IR light can be seen when pointing a camera at it.

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by zurohki** [comment]
With my original Vive base stations, you can manually update them on Linux by copying the firmware file over, it just doesn't work. I soft bricked mine - they didn't run at all, but uploading a new firmware file still worked. I recovered them by plugging them into a Windows machine and updating the firmware there.

Just saying, be careful fiddling with them. I'd update them on a Windows machine until updating base stations is officially supported.

**Comment by kedodrill** [comment]
You can't update the base stations, as it entirely relies on Bluetooth.

https://github.com/ValveSoftware/SteamVR-for-Linux/issues/207

Above is a more generalized issue, so thank you @jug007 for making this specific issue.  Hopefully we will be able to update our base stations soon :)

**Comment by yaomtc** [comment]
As of now the update system is partially implemented, but it crashes in the process of trying to update. Fortunately, it detects this the next time I start SteamVR and does a successful recovery (not sure if any firmware data was sent to the device at any point, but the base station doesn't get bricked, so that's nice)

**Comment by johnblommers** [comment]
It's February 2024 and my base stations cannot be updated on Linux. This running the very latest Valve software.

**Comment by yaomtc** [comment]
When you comment, it comes with a timestamp attached. But thanks for the update.

**Comment by johnblommers** [comment]
> When you comment, it comes with a timestamp attached. But thanks for the update.

Yes, comments do indeed come with time stamps. By also stating the date I was adding emphasis;  repeating a point to ensure the message is understood. 

Which is  that users are languishing waiting for a 100% working Valve VR solution.

**Comment by 8bitprodigy** [comment]
It's December 11th and this issue still persists. What the hell is Valve even doing?

**Comment by Hicks79** [comment]
Same here.

https://github.com/ValveSoftware/SteamVR-for-Linux/issues/755
https://github.com/ValveSoftware/SteamVR-for-Linux/issues/207

**Comment by johnblommers** [comment]
> It's December 11th and this issue still persists. What the hell is Valve even doing?

What Valve is doing is pushing its loyal Linux-using customers towards Microsoft Windows and Meta Quest products. That is a crying shame.

Gabe, WTF?
